src/js/translator-mode.js

```diff
@@ -286,4 +286,4 @@ function getCaretCharOffset(element) {
   return caretOffset;
 }
 
-const langTag = () => autoPrefixTag.get().replace(/\$filterLang/gi, languageNameCode[language.get()].code);
+const langTag = () => languages.length <= 0 ? autoPrefixTag.get() : autoPrefixTag.get().replace(/\$filterLang/gi, languageNameCode[languages.get()[0]].code);
```

Comment:
Instead of `languageNameCode[languages.get()[0]]`, use the repurposed `language` store.